updates/metalink                                         | 2.1 kB     00:00     
Setting up Install Process
Resolving Dependencies
--> Running transaction check
---> Package python-dateutil.noarch 0:1.4.1-4.fc12 set to be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                Arch          Version               Repository     Size
================================================================================
Installing:
 python-dateutil        noarch        1.4.1-4.fc12          fedora        227 k

Transaction Summary
================================================================================
Install       1 Package(s)

Total download size: 227 k
Installed size: 504 k
Downloading Packages:
http://ftp.heanet.ie/pub/fedora-archive/fedora/linux/releases/12/Everything/i386/os/Packages/python-dateutil-1.4.1-4.fc12.noarch.rpm: [Errno 14] HTTP Error 403 : http://ftp.heanet.ie/pub/fedora-archive/fedora/linux/releases/12/Everything/i386/os/Packages/python-dateutil-1.4.1-4.fc12.noarch.rpm 
Trying other mirror.
python-dateutil-1.4.1-4.f 63% [=========-     ]  0.0 B/s | 144 kB     --:-- ETA python-dateutil-1.4.1-4.fc12.noarch.rpm                  | 227 kB     00:00     
Running rpm_check_debug
Running Transaction Test
Transaction Test Succeeded
Running Transaction
  Installing     : python-dateutil-1.4.1-4.fc [                           ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [#                          ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [##                         ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [####                       ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [#####                      ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [######                     ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [########                   ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [##########                 ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [##############             ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [#################          ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [###################        ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [####################       ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [#######################    ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [#########################  ] 1/1  Installing     : python-dateutil-1.4.1-4.fc [########################## ] 1/1  Installing     : python-dateutil-1.4.1-4.fc12.noarch                      1/1 

Message from syslogd@planetlab1 at Apr 22 18:56:57 ...
 kernel:unregister_netdevice: waiting for tap1556-1 to become free. Usage count = 1

Installed:
  python-dateutil.noarch 0:1.4.1-4.fc12                                         

Complete!
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 13  8320   13  1095    0     0   3632      0  0:00:02 --:--:--  0:00:02  7348100  8320  100  8320    0     0  18414      0 --:--:-- --:--:-- --:--:-- 27826
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0107  1288  107  1288    0     0   4290      0 --:--:-- --:--:-- --:--:--  8644
Restarting experimentd...rm: cannot remove `nohup.out': No such file or directory
Done
Connection to planetlab1.di.fct.unl.pt closed.
